.buildkite/scripts/pipelines/pull_request/pipeline.ts

```diff
@@ -139,6 +140,9 @@ const uploadPipeline = (pipelineContent: string | object) => {
       !GITHUB_PR_LABELS.includes('ci:skip-cypress-osquery')
     ) {
       pipeline.push(getPipeline('.buildkite/pipelines/pull_request/osquery_cypress.yml'));
+      pipeline.push(
```

Comment:
after me move it to the single file it will be not needed

Reply:
Oh, missed your previous comment 👍